sig-platform/community-donated-plugins.md

```diff
@@ -0,0 +1,13 @@
+# Community Donated Plugins
+
+The [`spinnaker-plugins`](https://github.com/spinnaker-plugins) GitHub organization is a holding repository for unofficial, community-contributed Spinnaker plugins.
+Typical cases for moving a plugin to this organization would include companies who require CLAs for contributions to their main GitHub organizations, but do not want to include that overhead for contributorship to the plugin itself. **Donating a plugin to the `spinnaker-plugins` organization is never required.**
+
+Plugin authors looking to transfer ownership of their plugins to the `spinnaker-plugins` organization will:
+
+- Continue to be the maintainers of their plugin.
+- Will be granted `write` access to the repository.
```

Comment:
I suppose updates to the list of maintainers will also be handled through the TOC members. do we need a clarifying phrase for it or is it assumed?